scripts/prepare_release.py

```diff
@@ -6,9 +6,30 @@
 
 from git import Repo
 from changelog import get_changelog, get_relevant_commits
+from packit.config.package_config import get_local_package_config
+from packit.exceptions import PackitConfigException
+from packit.utils.specfile_version import set_specfile_version
```

Comment:
I am not sure if I like that we use Packit here. Can't we move the code to specfile?

Also not sure about parsing Packit config even there is a exception catch for that.

Reply:
> I am not sure if I like that we use Packit here. Can't we move the code to specfile?

Good point, it could be useful to have it in there.

> Also not sure about parsing Packit config even there is a exception catch for that.

We could make those options of the GH action, but then you'd have to configure it in two places.